integr8ly/tutorial-web-app
# INTLY-3511: Use shared Fuse for walkthroughs

## What
Use the shared fuse provisioned by the integreatly operator for walkthrough 1A instead of provisioning Fuse in the user namespace.

## Why
By doing this, we won't need to add the pull secret to the walkthrough namespaces and no extra permissions are needed to be provided to the dev user for creating subscriptions.

## Verification Steps
1. Run the webapp and point it to an OpenShift 4 cluster.
   - Pass the following env vars when running `yarn start:dev`
   - `OPENSHIFT_OAUTHCLIENT_ID`, `OPENSHIFT_OAUTH_HOST`, `OPENSHIFT_API`,  `OPENSHIFT_HOST`, `SSO_ROUTE`, `OPENSHIFT_VERSION='4'`, `INSTALLED_SERVICES`
2. Go to Walkthrough 1.
3. Click on `Get Started`
   - Verify: Walkthrough 1A starts successfully.
4. Go through Walkthrough 1A.
   - Verify: All the links to Fuse should be directed to the shared Fuse provisioned by the Integreatly operator.

## Checklist:

## Additional Notes
Related JIRA - https://issues.jboss.org/browse/INTLY-3511
